### ISSUE TITLE
Any plans for DSP support? (altering the data before playing and/or encoding)

### ISSUE THREAD

**Issue by sp-1234** [andrewrk/libgroove#135]
I didn't see any mention in README so I decided to ask, if it would be possible in some future, before I start rolling my own [NIH](https://en.wikipedia.org/wiki/Not_invented_here) player 😄

**Comment by thejoshwolfe** [comment]
What do you mean by DSP support? Do you a standard library that uses cpu-specific dsp assembly instructions?

**Comment by magicgoose** [comment]
No, not necessarily so. I mean simply having possibility to do some audio processing on the fly when playing or transcoding, like EQ, [crossfeed](https://en.wikipedia.org/wiki/Crossfeed#Digital_.28DSP.29), etc. A typical example of audio player which can do that is foobar2000.

**Comment by thejoshwolfe** [comment]
Oh i'm sorry. I thought this issue was in a different repo. derp. Your question makes a lot more sense now.
